### PR TITLE
feature - create favorite Screen

### PR DESCRIPTION
**Pull request by at-haivo**
create favorite Screen

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: Youtube/Models/Video.swift [hunk 1]
```diff
@@ -13,11 +13,13 @@ class Video: NSObject {
     var title: String = ""
     var thumbnailPath: String?
     var timeDescription: String?
```
Comment (at-daiho) [inline Youtube/Models/Video.swift]: time: String?

File: Youtube/Models/Video.swift [hunk 2]
```diff
@@ -13,11 +13,13 @@ class Video: NSObject {
...
+    var videoDescription: String?
```
Comment (at-daiho) [inline Youtube/Models/Video.swift]: description: String?

File: Youtube/Controllers/FavoriteVC/FavoriteViewController.swift [hunk 3]
```diff
@@ -7,14 +7,92 @@
...
+    func displayAlertMessage(messageToDisplay: String) {
+        let alertController = UIAlertController(title: "Alert", message: messageToDisplay, preferredStyle: .alert)
+        let okAction = UIAlertAction(title: "OK", style: .default) { ( _ ) in
+            self.videos.removeAll()
+            self.tableView.reloadData()
+        }
+        let cancelAction = UIAlertAction(title: "Cancel", style: .cancel, handler: nil)
+        alertController.addAction(okAction)
+        alertController.addAction(cancelAction)
+        self.present(alertController, animated: true, completion:nil)
```
Comment (at-daiho) [inline Youtube/Controllers/FavoriteVC/FavoriteViewController.swift]: remove self.

File: Youtube/Controls/Cells/FavoriteTableViewCell.swift [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
+    }
+
+    func config(video: Video) {
+        if let image = video.thumbnailPath {
+            thumbnailImageView.image = UIImage(named: image)
+        }
```
Comment (at-daiho) [inline Youtube/Controls/Cells/FavoriteTableViewCell.swift]: Please add else or prepairForReuse

**Comment by at-haivo** [comment]
please review fix comment

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: Youtube/Controllers/HomeVC/cell/VideoCollectionViewCell.swift [hunk 5]
```diff
@@ -10,14 +10,14 @@ import UIKit
 
 class VideoCollectionViewCell: UICollectionViewCell {
 
-    @IBOutlet weak var thumbnailImageView: UIImageView!
-    @IBOutlet weak var timeLabel: UILabel!
+    @IBOutlet private weak var thumbnailImageView: UIImageView!
+    @IBOutlet private weak var timeLabel: UILabel!
 
-    func config(video: Video) {
+    func add(video: Video) {
```
Comment (at-daiho) [inline Youtube/Controllers/HomeVC/cell/VideoCollectionViewCell.swift]: func config(video: Video)

File: Youtube/Controls/Cells/FavoriteTableViewCell.swift [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    @IBOutlet private weak var timeLabel: UILabel!
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var descriptionLabel: UILabel!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
+    }
+
+    func add(video: Video) {
```
Comment (at-daiho) [inline Youtube/Controls/Cells/FavoriteTableViewCell.swift]: func config(video: Video)

**Pull request merged**